DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.5_Master_Method_Video.md

```diff
@@ -0,0 +1,19 @@
+# Master Method
+
+This method is used to solve recurrence relations of the form:
+
+_T(n) = aT(n/b) + $\theta$ ($n^k log^pn$) where a $\geq$ 1, b > 1, k $\geq$ 0 and p $\in$ R_
```

Comment:
Fix this $\theta$ ($n^k log^pn$) where a $\geq$ 1, b > 1  and  rest is looking good.

Reply:
The output looks like this - 
![recur](https://user-images.githubusercontent.com/68556975/134801306-2741f925-c2c8-47c7-b13d-df9a8936081b.JPG)
Won't this be easier for readers to read?